### DISCUSSION TITLE
Pimcore Copilot not working for Enterprise version

### DISCUSSION THREAD

**Discussion by mano-dharmalingam** [pimcore/pimcore#17215] [Q&A / Support]
Hi,
i have installed Pimcore Enterprise version and also installation of copilot is completed but the ALT + X not working and also when i tried to see the bundle list it is showing that copilot is enabled but not installed. i followed below link.

https://pimcore.com/docs/platform/Copilot/Installation

**Comment by fashxp** [comment]
Did you try to install it again?
Any error messages?
